cmd/run.go

```diff
@@ -323,6 +334,9 @@ a commandline interface for interacting with it.`,
 			logger.Debug("Waiting for engine processes to finish...")
 			engineWait()
 			logger.Debug("Everything has finished, exiting k6!")
+			if interrupt != nil {
+				return errext.WithExitCodeIfNone(interrupt, exitcodes.ScriptException)
```

Comment:
To clarify: you're suggesting to use a different name for exit code `107` or a different exit code for when the test is aborted by the script? Changing the name is fine, but changing the exit code will require also changing k6agent to handle it properly, and I don't see why we should since `107` is only used for this case when the test is interrupted.

Reply:
I am proposing that  `AbortedByScriptErrorCode` (name subject to change) is added and have it have a *new* exit code so we can differentiate between some using `abort()` and there is an exception in the init context

Reply:
I settled on `ScriptAborted` (108) in d46a401a, and simplified some things in `cmd/run.go`. Let me know.